### PR TITLE
getPortfolios

### PR DESCRIPTION
**Pull request by Mejiao99**

### CODE REVIEW

File: HelloWorldFunction/src/main/java/helloworld/App.java [hunk 1]
```diff
@@ -5,12 +5,14 @@
 import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
 import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import helloworld.Portfolio.OperationRequest;
```
Comment (jmena) [inline HelloWorldFunction/src/main/java/helloworld/App.java]: ?

File: HelloWorldFunction/src/main/java/helloworld/App.java [hunk 2]
```diff
@@ -26,16 +28,11 @@ public APIGatewayProxyResponseEvent handleRequest(final APIGatewayProxyRequestEv
 
         APIGatewayProxyResponseEvent response = new APIGatewayProxyResponseEvent()
                 .withHeaders(headers);
+        PortfolioRequester generator = new PortfolioRequester();
```
Comment (jmena) [inline HelloWorldFunction/src/main/java/helloworld/App.java]: 1. Rename `generator` to `portfolioProvider`.

File: HelloWorldFunction/src/main/java/helloworld/Portfolio/PortfolioRequester.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package helloworld.Portfolio;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+
+public class PortfolioRequester implements PortfoliosGenerator {
```
Comment (jmena) [inline HelloWorldFunction/src/main/java/helloworld/Portfolio/PortfolioRequester.java]: Rename `PortfolioRequester` to `PortfolioProvider`

File: HelloWorldFunction/src/test/java/helloworld/AppTest.java [hunk 4]
```diff
@@ -1,20 +1,21 @@
 package helloworld;
```
Comment (jmena) [inline HelloWorldFunction/src/test/java/helloworld/AppTest.java]: Move to a package.

File: HelloWorldFunction/src/main/java/helloworld/Portfolio/PortfolioRequester.java [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+    @Override
+    public List<Portfolio> getPortfolios(OperationRequest request) {
```
Comment (jmena) [inline HelloWorldFunction/src/main/java/helloworld/Portfolio/PortfolioRequester.java]: Add some unit tests.

File: HelloWorldFunction/src/main/java/helloworld/App.java (hunk 2 above)
Comment (Mejiao99) [reply HelloWorldFunction/src/main/java/helloworld/App.java]: Solved in the next commit

File: HelloWorldFunction/src/main/java/helloworld/Portfolio/PortfolioRequester.java (hunk 3 above)
Comment (Mejiao99) [reply HelloWorldFunction/src/main/java/helloworld/Portfolio/PortfolioRequester.java]: Solved in the next commit

File: HelloWorldFunction/src/test/java/helloworld/AppTest.java (hunk 4 above)
Comment (Mejiao99) [reply HelloWorldFunction/src/test/java/helloworld/AppTest.java]: Resolved in next commit

File: HelloWorldFunction/src/main/java/helloworld/App.java (hunk 1 above)
Comment (Mejiao99) [reply HelloWorldFunction/src/main/java/helloworld/App.java]: moved package

File: HelloWorldFunction/src/main/java/helloworld/Portfolio/PortfolioRequester.java (hunk 5 above)
Comment (Mejiao99) [reply HelloWorldFunction/src/main/java/helloworld/Portfolio/PortfolioRequester.java]: added on next commit

**Pull request merged**